.taskcluster.yml

```diff
@@ -26,7 +26,7 @@ tasks:
 
     scopes: [
       "queue:create-task:lowest:{{ taskcluster.docker.provisionerId }}/deepspeech-worker",
-      "queue:create-task:lowest:{{ taskcluster.docker.provisionerId }}/deepspeech-win",
+      "queue:create-task:lowest:{{ taskcluster.docker.provisionerId }}/deepspeech-win-b",
```

Comment:
same comment as on tensorflow repo, maybe we should just update `deepspeech-win` instances